app/models/entry.server.ts

```diff
@@ -50,6 +49,46 @@ export function getEntries(skip?: number, take: number = 20) {
   })
 }
 
+export async function putEntries(data: any) {
```

Comment:
Let's avoid naming it `putEntries` (unless there's a convention I'm missing!) when it creates an entry. `put` is typically associated with the REST `PUT` verb, which refers to an update of an existing entry. Something like `create` would probably be better.

Also, can this code actually create multiple entries, or does it just create one at a time?

Reply:
yeah good point. I changed it to "insertEntry." Though "create" actually makes a lot of sense in hindsight